components/SectionTitle.tsx

```diff
@@ -16,16 +16,20 @@ export default function SectionTitle(props: Props) {
   const router = useRouter()
   const path = router.asPath.split('#')[0]
   const id = `section-${props.index}`
+  const generated = new Date()
 
   return (
-    <h3
-      className="mb-8 text-2xl font-bold border-b-2 border-black pb-2"
-      id={id}
-    >
-      {props.icon && <span className="mr-2">{props.icon}</span>}
-      <Link href="/reports/[name]" as={`${path}#${id}`}>
-        <a>{props.children}</a>
-      </Link>
-    </h3>
+    <div className="mb-8 border-b-2 border-black">
+      <h3 className="mb-2 text-2xl font-bold" id={id}>
+        {props.icon && <span className="mr-2">{props.icon}</span>}
+        <Link href="/reports/[name]" as={`${path}#${id}`}>
+          <a>{props.children}</a>
+        </Link>
+      </h3>
+
+      <p className="mb-2 font-normal text-gray-400 italic text-sm">
+        Generated {generated.toLocaleString()}
+      </p>
```

Comment:
I moved this and de-emphasized it visually @bryanmacfarlane 

<img width="451" alt="Screen Shot 2020-09-18 at 11 29 00 AM" src="https://user-images.githubusercontent.com/412282/93616109-4a03c280-f9a2-11ea-81b9-b3d7fede318a.png">

You can use [Tailwind CSS margin classes](https://tailwindcss.com/docs/margin#app) for vertical spacing like this. Gives you a bit more control than a `<br />`. There's also a VSCode IntelliSense plugin for it that's _amazing_.